spring-cloud-deployer-kubernetes/src/main/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesDeployerProperties.java

```diff
@@ -58,6 +58,11 @@ public class KubernetesDeployerProperties {
      */
     private int maximumConcurrentTasks = 20;
 
+	/**
```

Comment:
These files all need fully reformatting - I will do that in a separate PR so that we can see the forest through the trees in this PR.